cache/local_fs.go

```diff
@@ -0,0 +1,63 @@
+package cache
+
+import (
+	"os"
+	"path/filepath"
+)
+
+type LocalFsCache struct {
+	cacheDir string
+}
+
+const (
+	DefaultCacheDir = "/tmp/risu/cache"
+)
+
+func NewLocalFsCache() Cache {
+	var cacheDir string
+
+	if os.Getenv("RISU_CACHE_DIR") != "" {
+		cacheDir = os.Getenv("RISU_CACHE_DIR")
+	}
+
+	if cacheDir == "" {
+		cacheDir = DefaultCacheDir
+	}
+
+	if _, err := os.Stat(cacheDir); err != nil {
+		os.MkdirAll(cacheDir, 0755)
+	}
+
+	return &LocalFsCache{cacheDir}
+}
+
+func (c *LocalFsCache) Get(key string) (string, error) {
+	cache := cachePath(key)
+	inflateDir := inflateDirPath(key)
+
+	if _, err := os.Stat(cache); err != nil {
+		return "", nil
```

Comment:
let's define our own error like [this](https://github.com/coreos/etcd/blob/master/error/error.go) later